docs/blogs/kubecon_cfp.md

```diff
@@ -0,0 +1,45 @@
+# ElasticDL: Kubernetes-native Deep Learning System with Elastic Scheduling
+
+## Description
+
+In these years, Kubernetes becomes the fundamental infrastructure of machine
+learning. Kubeflow project provides some operators to run distributed machine
+learning jobs on Kubernetes. It uses gang scheduling policy. For example, a job
+requires N pod, it will keep pending if there are N-1 or less available pods. And
+during execution, the job will crash if any one of these N instances crashes. As
+a result, the job pending time is long and the resouce utilization is insufficient.
+
+ElasticDL implements elastic scheduling policy. A job doesn't need wait for all
+the required resource ready and start promptly. A job process can be preempted
+by a higher priority job, join freely once the resource become sufficient later
+and the entire job keeps running. It can significantly shorten the job waiting
+time and improve the resource utilization.
+
+## Benefits to the Ecosystem
+
+This talk introduces a system ElasticDL, it implements elastic scheduling policy
+which is new to deep learning workloads scheduling on Kubernetes. Compared with
+the gang scheduling policy that kubernetes operators from Kubeflow apply, it
+can improve the resource utilization significantly. We add a master to launch,
+monitor and manage the worker pods using Kubernetes API and distribute the
+sharded data dynamically to them. It doesn't rely on the Kubernetes operators.
+In this way, master can understand not only the internal details of deep
+learning process but also the model structure, and then make more optimized
+scheduling decision.
+
+For system developers and operators, this talk introduces a elastic scheduling
```

Comment:
Deep learning researchers like ElasticDL as it reduces the pending time of each job as it makes full use of residual resources to run as many concurrent experiments as possible.  Deep learning jobs depend on many parameters, including the optimizer, activator, and cost, hyperparameters.  Users are eager to see the status of the first few iterations of the learning job so to ensure the configuration is mathematically correct.  ElasticDL provides a cure.

Using the residual resource, ElasticDL also shortens the total time to run a batch of training jobs.

Reply:
Good suggestion, updated according to the comments.